File: src/Reference.CSharp/Migrations/20190520_InitialCreate.cs
using Microsoft.EntityFrameworkCore.Migrations;
using System;
// ReSharper disable RedundantExtendsListEntry
// ReSharper disable PartialTypeWithSinglePart
// ReSharper disable RedundantArgumentDefaultValue
// ReSharper disable ArgumentsStyleAnonymousFunction
// ReSharper disable ArgumentsStyleStringLiteral

namespace Reference.CSharp.Migrations
{
    public partial class InitialCreate : Migration
    {
        protected override void Up(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.CreateTable(
                name: "GeoData",
                columns: table => new
                {
                    Uuid = table.Column<Guid>(nullable: false),
                        //.Annotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn),
                    Latitude = table.Column<double>(nullable: true),
                    Longitude = table.Column<double>(nullable: true),
                    PlaceId = table.Column<long>(nullable: true),
                    DisplayName = table.Column<string>(nullable: true),
                    City = table.Column<string>(nullable: true),
                    County = table.Column<string>(nullable: true),
                    State = table.Column<string>(nullable: true),
                    Country = table.Column<string>(nullable: true),
                    CountryCode = table.Column<string>(nullable: true)
                },
                constraints: table =>
                {
                    table.PrimaryKey("PK_GeoData", x => x.Uuid);
                });
        }

        protected override void Down(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.DropTable(name: "GeoData");
        }
    }
}

File: src/Reference.VB/Migrations/20190520_InitialCreate.vb
Imports Microsoft.EntityFrameworkCore.Migrations

Namespace Migrations
    Partial Public Class InitialCreate
        Inherits Migration

        Protected Overrides Sub Up(migrationBuilder As MigrationBuilder)

            migrationBuilder.CreateTable(name:="GeoData",
                columns:=Function(table)
                             Return New With {
                                .Uuid = table.Column(Of Guid)(nullable:=False),
                                .Latitude = table.Column(Of Double)(nullable:=True),
                                .Longitude = table.Column(Of Double)(nullable:=True),
                                .PlaceId = table.Column(Of Long)(nullable:=True),
                                .DisplayName = table.Column(Of String)(nullable:=True),
                                .City = table.Column(Of String)(nullable:=True),
                                .County = table.Column(Of String)(nullable:=True),
                                .State = table.Column(Of String)(nullable:=True),
                                .Country = table.Column(Of String)(nullable:=True),
                                .CountryCode = table.Column(Of String)(nullable:=True)
                            }
                         End Function,
                constraints:=Sub(table)
                                 table.PrimaryKey("PK_GeoData", Function(x) x.Uuid)
                             End Sub)

        End Sub

        Protected Overrides Sub Down(migrationBuilder As MigrationBuilder)
            migrationBuilder.DropTable("GeoData")
        End Sub

    End Class

End Namespace
